content/applications/hr/fleet/configuration.rst

```diff
@@ -57,109 +57,129 @@ for existing available cars in this field.
+| A4    | Serie 5      | Class C   | Antara        |
++-------+--------------+-----------+---------------+
+| A5    | Serie 6      | Class CL  | Astra         |
++-------+--------------+-----------+---------------+
+| A6    | Serie 7      | Class CLS | AstraGTC      |
++-------+--------------+-----------+---------------+
+| A7    | Serie Hybrid | Class E   | Combo Tour    |
++-------+--------------+-----------+---------------+
+| A8    | Serie M      | Class GL  | Corsa         |
++-------+--------------+-----------+---------------+
+| Q3    | Serie X      | Class GLK | Insignia      |
++-------+--------------+-----------+---------------+
+| Q5    | Serie Z4     | Class M   | Meriva        |
++-------+--------------+-----------+---------------+
+| Q7    |              | Class R   | Mokka         |
++-------+--------------+-----------+---------------+
+| TT    |              | Class S   | Zafira        |
++-------+--------------+-----------+---------------+
+|       |              | Class SLK | Zafira Tourer |
++-------+--------------+-----------+---------------+
+|       |              | Class SLS |               |
++-------+--------------+-----------+---------------+
 
 .. _fleet/configuration/add-model:
 
 Add a new model
----------------
+~~~~~~~~~~~~~~~
 
 New vehicle models can easily be added to the database. To add a new model, navigate to
-:menuselection:`Fleet app --> Configuration --> Vehicle Models`. Click :guilabel:`Create` and a
-vehicle model form will load. Enter the following information on the form, then click
-:guilabel:`Save`. Be advised, some fields are specific to Belgian based companies, so not all fields
-or sections may be visible depending on the location of the company.
+:menuselection:`Fleet app --> Configuration --> Models: Models`. Click the :guilabel:`New` button in
+the top-left corner and a blank vehicle model form loads.
+
+Enter the following information on the form:
+
+.. note::
+   Some fields or sections are based on the localization setting of the company, therefore not all
+   fields or sections may be visible.
 
 - :guilabel:`Model name`: enter the model name in the field.
-- :guilabel:`Manufacturer`: select the manufacturer from the drop-down menu. If the manufacturer is
-  not configured, type in the manufacturer and then click :guilabel:`Create` or :guilabel:`Create &
-  Edit`.
+- :guilabel:`Manufacturer`: select the manufacturer from the drop-down menu. If a logo is configured
+  for the selected manufacturer, the logo automatically loads in the image box in the top right
+  corner.
+
+  If the manufacturer is not already configured, type in the manufacturer name and then click
+  :guilabel:`Create` to add the manufacturer, or :guilabel:`Create and edit...` to :ref:`add the
+  manufacturer and the logo <fleet/configuration/add-manufacturer>` in a :guilabel:`Create
+  Manufacturer` pop-up window.
 - :guilabel:`Vehicle Type`: select one of two preconfigured vehicle types, either :guilabel:`Car` or
```

Comment:
Can't add the suggestion to line 170-
"preconfigure". You use pre-configure" with a hyphen throughout the doc. This is one of those where both are *technically* correct, but we should probably talk about this for the style guide. For now, I would say choose one and be consistent in this doc. :smile:

Reply:
I didn't realize I switched- thanks for catching that! I tried to be consistent, but didn't get them all. And yeah, we should add this to the guide as well - I agree =)